src/analysis/retail/monk/mistweaver/modules/spells/Vivify.tsx

```diff
@@ -293,6 +341,14 @@ class Vivify extends Analyzer {
         @ <strong>{this.owner.formatTimestamp(event.timestamp)}</strong>, ReMs:{' '}
         <strong>{rems}</strong>
         <br />
+        <>
+          Full Overheal Hits: {fullOverhealHits} ({fullOverhealHits}/{rems} hits)
```

Comment:
```suggestion
          Full Overheal Hits: {fullOverhealHits} (avg {fullOverhealHits}/{rems} hits)
```
I assume you're doing an average overheal hits here

Reply:
No this is a flat metric of how many vivify cleave hits were completely useless and/or actively detrimental to your effective healing, and helps give more indication on the QP color, for example: 
![image](https://github.com/WoWAnalyzer/WoWAnalyzer/assets/26779541/90f1d138-534c-492e-a162-6381ec69d7a3)
Felt this was necessary since its a stark change from last patch where more rem = more better. I will also update the QP explanation text for the section